CorsixTH/Src/th_gfx_font.cpp

```diff
@@ -653,8 +653,16 @@ text_layout freetype_font::draw_text_wrapped(render_target* pCanvas, const char*
             }
 
             // Determine if a line can be broken at the current position.
-            if(iCode == ' ')
+            if (iCode == ' ') {
                 sLineBreakPosition = sOldMessage;
+            } else if (iCode == 0x3002 ||
+                    iCode == 0xff0c ||
+                    iCode == 0xff0d ||
+                    iCode == 0xff1b ||
+                    iCode == 0xff1f) {
+
+                sLineBreakPosition = sOldMessage + 3; // break after this codepoint (3 bytes in utf-8)
```

Comment:
I assume the `' '` matched at 656 gets removed (by the looks of it at 633?), so would that mean the character after the line break match you set here also gets removed?